tests/improver-nbhood/00-null.bats

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bats
+
+@test "nbhood no arguments" {
```

Comment:
It seems like 00-help.bats, 01-help.bats and 01-bad-option.bats have been added, which all test that the desired help statement has been produced when requested. These tests seem fine.